source/agora/consensus/validation/Block.d

```diff
@@ -179,6 +181,8 @@ public string isGenesisBlockInvalidReason (const ref Block block) nothrow @safe
     UTXO[Hash] utxo_set;
     foreach (const ref tx; block.txs)
     {
+        // Data storage type(TxType.Payload) is currently not required for Genesis Block,
```

Comment:
It would be better to explicitly disallow payload transactions in the genesis block.